### PR TITLE
Use LaTeX to show the galgebra logo in the top left

### PR DESCRIPTION
**Pull request by eric-wieser**
image

Albeit briefly at startup

image

### CODE REVIEW

**Comment by utensil** [comment]
How about making it a static svg generated from LaTeX and add it as an image with `alt` set to `GAlgebra`

**Comment by utensil** [comment]
I've been using https://viereck.ch/latex-to-svg/ for that

**Comment by eric-wieser** [comment]
Could do, but then you lose that nifty selection ability:

image

File: doc/_templates/layout.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  {% if logo and theme_logo_only %}
+    <a href="{{ pathto(master_doc) }}">
+  {% else %}
+    <a href="{{ pathto(master_doc) }}" class="icon icon-home" alt="{{ _("Documentation Home") }}"> {{ project }}
+  {% endif %}
+
+  {# this div is instead of the usual logo. The height is picked to avoid a jump
+     when mathjax renders #}
+  <div class="math notranslate nohighlight" style="height: 4.4em; display: table">
```
Comment (utensil) [inline doc/_templates/layout.html:12]:
```suggestion
  <div class="math notranslate nohighlight" style="height: 4.4em; display: table; color: rgba(0, 0, 0, 0)">
```

Comment (utensil) [reply doc/_templates/layout.html:12]: This could avoid the flashing by hiding the source code and it has no effect on the white MathJax.

Comment (eric-wieser) [reply doc/_templates/layout.html:12]: Clever. I assume `color: transparent` works too?

Comment (eric-wieser) [reply doc/_templates/layout.html:12]:
```suggestion
  <div class="math notranslate nohighlight" style="height: 4.4em; display: table; color: transparent">
```

Comment (utensil) [reply doc/_templates/layout.html]: Need some tests to confirm.

Comment (eric-wieser) [reply doc/_templates/layout.html]: I think it's working now?

**Comment by eric-wieser** [comment]
`transparent` does the same as `rgba`. Want to go ahead with this, or not?

**Pull request merged**